### ISSUE TITLE
object-curly-spacing with never option warns for default and multiple named imports

### ISSUE THREAD

**Issue by lukekarrys** [eslint/eslint#3370]
I added the following test (after cloning master) to the `valid` array in `tests/lib/rules/object-curly-spacing` and ran `npm test`.

**Test**
```js
{
  code: "import door, {house, mouse} from 'caravan'",
  options: ["never"],
  ecmaFeatures: { modules: true }
}
```

**Expected**

The tests would pass.

**Actual**
```
  1) object-curly-spacing import door, {house, mouse} from 'caravan':

      AssertionError: Should have no errors but had 1: [ { ruleId: 'object-curly-spacing',
    severity: 1,
    message: 'There should be no space after \',\'',
    line: 1,
    column: 20,
    nodeType: 'ImportDeclaration',
    source: 'import door, {house, mouse} from \'caravan\'' } ]
```

It warns for the comma between `house` and `mouse` which I believe shouldn't be a warning since there is already a passing test for `import {house, mouse} from 'caravan'`.

**Comment by eslintbot** [comment]
Thanks for the issue! We get a lot of issues, so this message is automatically posted to each one to help you check that you've included all of the information we need to help you.

Reporting a bug? Please be sure to include:

1. The version of ESLint you are using (run `eslint -v`)
1. The source code that caused the problem
1. The configuration you're using (for the rule or your entire config file)
1. The actual ESLint output complete with line numbers

Requesting a new rule? Please be sure to include:

1. The use case for the rule - what is it trying to prevent or flag?
1. Whether the rule is trying to prevent an error or is purely stylistic
1. Why you believe this rule is generic enough to be included

Requesting a feature? Please be sure to include:

1. The problem you want to solve (don't mention the solution)
1. Your take on the correct solution to problem

Including this information in your issue helps us to triage it and get you a response as quickly as possible.

Thanks!

**Comment by lukekarrys** [comment]
If this is reviewed as an actual bug, I'd be interested in attempting to fix it with a pull request. :smile:

**Comment by nzakas** [comment]
Definitely looks like a bug. Seems like it's checking the comma for some reason.

**Comment by lukekarrys** [comment]
I'm taking a look at this and should have a pull request ready in the next few days.
